contrib/bmp2/README.md

```diff
@@ -0,0 +1,38 @@
+BMP2
```

Comment:
I know this hardware existed for a very long time. I wish back then we asked you to name this differently. BMPM V2.x also exists and BMP2 is very easy to confuse for the users. Any thoughts about this @emeb?

Reply:
Yeah - naming it anything BMP* was not very neighborly. I should have been more imaginative about it back when, but then coming up with project names has never been a strength for me. Kind of late to be doing anything about it now, but if you'd like to modify the name in the repo that would be fine with me.

Reply:
Yeah all I am asking for is to rename it in the repo, to address the name collision. :) You don't even have to rename the assets internally in the design files. But the readme should address the fact that it is named differently internally.

Reply:
I will use his username as folder name (as contributor)

Reply:
Once you create the new folder w/ my username would it be good for me to create a README.md within that folder that essentially includes all the info + assets from the outside source?